src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -62,7 +63,8 @@ export default class DataTable extends Vue {
 
   _head() {
     const tableHeadCells = [
-      this.config.selectable ? this._selectRowCheckbox(true) : null,
+      this.config.selectable && this.config.selectable !== 'radio' ? this._selectRowCheckbox(true) : null,
+      this.config.selectable === 'radio' ? this._radioButton(true) : null,
```

Comment:
```suggestion
      this.config.selectable
        ? this.config.selectable === 'radio'
          ? this._radioButton(true)
          : this._selectRowCheckbox(true)
        : null,
```
Please, try this. I think it's cleaner and more readable.